mergify_engine/context.py

```diff
@@ -371,10 +386,39 @@ async def iter_mergify_config_files(
                 ),
             )
 
+    async def get_mergify_config(self) -> "rules.MergifyConfig":
+        # circular import
+        from mergify_engine import rules
```

Comment:
why do we need to import it here since it is mentionned here line 57 ?
`if typing.TYPE_CHECKING:
    from mergify_engine import rules`

Reply:
Fixed